analyzer-comments/elixir/accumulate/use_recursion.md

```diff
@@ -0,0 +1,3 @@
+# use recursion
+
+The purpose of this exercise is to teach recursion. Solve it without using list comprehensions or any of the functions from the modules `Enum`, `List`, or `Stream`.
```

Comment:
About bird-count it was true that its purpose is to _teach_ recursion because it was a concept exercise, but accumulate is a practice exercise, so let's rephrase:
```suggestion
The purpose of this exercise is to practice recursion. Solve it without using list comprehensions or any of the functions from the modules `Enum`, `List`, or `Stream`.
```